app/views/foreman_bootdisk/generic_boot_no_dhcp.erb

```diff
@@ -12,12 +12,16 @@
 <% (0..32).each do |i| -%>
 :net<%= i %>
 isset ${net<%= i -%>/mac} || goto no_nic
-echo net<%= i -%> is a ${net<%= i -%>/chip} with MAC ${net<%= i -%>/mac}
-echo -n Enter this interface name to boot from it (net<%= i -%>|n):  && read interface
-iseq ${interface} net<%= i -%> && goto get_static_ip
+echo Found ${net<%= i -%>/mac} as net<%= i -%> on a ${net<%= i -%>/chip}
 goto net<%= i+1 %>
 <% end -%>
 
+echo -n Enter the interface name to boot from based on above MAC addresses (e.g net0|n):  && read interface
```

Comment:
The text is quite long, remember it's only 79 columns. Can you make it shorter?

Also net0|n can be misleading, just "e.g. net0" is fine I guess.